web/help/GuideUnit1.md

```diff
@@ -517,6 +517,16 @@ letters of the function you want, you can narrow down the list.  The type
 signatures tell you what types of information you need to provide to apply
 the function and what type you can expect to end up with.
 
+**Important**: The names of functions are *reserved*. That means you cannot use them as variable names. You would not be able to write program like this:
```

Comment:
This would be a great time to use a warning callout box.  The syntax looks like this:

```
!!! Warning: Title goes here
    Text goes here.  Each line should all be indented 4 spaces.  You can put code blocks and everything.
```

I'm also not a huge fan of using "reserved" here.  I think there's an easier idea: a variable name can only refer to one thing.  Since the built-in functions are already defined by CodeWorld itself, students are not allowed to use the same name for their own variables.  This is exactly like how they cannot define the same variable twice in their own code.  There's no reason to make these different ideas, when they are really the same idea.